### ISSUE TITLE
Donations appreciated

### ISSUE THREAD

**Issue by cryptopoo** [cryptopoo/hiveos_gddr6x_temps#24]
Dear folks,

the script has now reached a very good stability and I'm very happy that thousands of you have actually downloaded and installed this.
It's amazing!

I'd like to renew here an invitation to donate if this work had a meaning to you. Big or small donation, everything is accepted as a token of appreciation :)

Thanks to all for bearing with me along the journey (that consisted so far in many days and nights of overtime), this wouldn't have been possible thanks to all of you!

```
BTC    bc1q0xrs5cq7zwrnvz7dkzvgrzya48se6zgkt90sln
ETH    0x9b7A231147c8294cC558F080B26F948Bfd523A79
ETC    0xe9c8F439196fBA21f383AE69e8E4330B1493C335
BNB    0x9b7A231147c8294cC558F080B26F948Bfd523A79
RVN    RB2JTzcSxfZtWcQhYMCgad2M1T8Xi7dgPX
ERGO   9h1DrdXqQP3Aq3rdAPtLH4n5g9mQrLL2W3pYe152CkqPKbimu6w
MATIC  0x9b7A231147c8294cC558F080B26F948Bfd523A79

ZIL    zil13ctkfhgx5tdp48sewtmddtujfgsl87cqz38jug
FLUX   t1HyF9wDivycxNLNRzs9rRoEDwJ6hogDjpd
TRX    TFRKg3FTRbcR79CnpahMzf5yJu1dU4FQGn
FTM    0x9b7A231147c8294cC558F080B26F948Bfd523A79
AE     ak_2ekWFyFHPazAxMHbdGrdd3emVHzDo9iprwAMCw1aaTYg6YZAn
FIRO   aBYiGagsNdZE71bb2uAKKGyixBu6195uE3
DASH   XuuZxRNiK9BzhCDHRci8XBCoiz5GN9xzig
```

Crypto Poo
:v: :v: :v: :v: :v: :v:

**Comment by dedizones** [comment]
Yes i finish solo Ergo and donate ERGO after man !

Hiveos patched same system Hiveon OS v0.6-214@220331 :/ 

SYSTEM 🖥
⚙️ Improved Nvidia MEM Temp reporting
    * added memory temperature for GDDR6X
    * fixed vendor detection for HBM2 memory
    
⚙️ Added support MEM Temp for Coolbox Autofan controller

It's same script or no ? hiveos contacted your for patch ?

**Comment by cryptopoo** [comment]
Hello, thanks! I will appreciate it :)

Yes I just saw that. I'm going to update now to see what they did.

They didn't contact me, in fact they probably have better ways to do it since my fix was sitting on top of HiveOS (so that the OS is untouched) while they can instead modify HiveOS to include a few lines to have this done :)

In principle they said they didn't want to do it, but I guess they changed idea now.

**Comment by cryptopoo** [comment]
Yep, so they get the info now directly from nvtool so my script is not needed anymore :)

**Comment by surfjaxwave** [comment]
Your script shows a4000 and theirs doesn't currently on my setup. It works on my a4500 and a2000.
![image](https://user-images.githubusercontent.com/102394321/161266083-0601e7f5-24e8-49dd-a6c7-11a18823f255.png)

**Comment by surfjaxwave** [comment]
Its actually working now. I noticed my fan was at 0 so I lowered overclocks and rebooted.


> Your script shows a4000 temp and HiveOs doesn't currently on my setup. HiveOs temp does work on my a4500 and a2000. So your script still needed ;)

**Comment by cryptopoo** [comment]
So your A4000 is correctly reporting temperatures, right? :)

**Comment by surfjaxwave** [comment]
> So your A4000 is correctly reporting temperatures, right? :)

yep :) The issue was the overclock stopped reporting the fan and temp. All good now.
